[krb5] Unable to build

```
../../util/et/compile_et -d ../../util/et --textdomain mit-krb5 et-c-ss_err.et
+ mawk -f ../../util/et/et_h.awk outfile=et-h-ss_err.h et-h-ss_err.et
mawk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
make[2]: *** [Makefile:501: std_rqs.c] Error 2
make[2]: *** Waiting for unfinished jobs....
mawk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
make[2]: *** [Makefile:473: ss_err.h] Error 2
+ mawk -f ../../util/et/et_h.awk outfile=et-c-ss_err.h et-c-ss_err.et
mawk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
make[2]: *** [Makefile:479: ss_err.c] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/krb5-1.16.tar.gz.dir/krb5-1.16/src/util/ss'
make[1]: *** [Makefile:863: all-recurse] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/krb5-1.16.tar.gz.dir/krb5-1.16/src/util'
make: *** [Makefile:1524: all-recurse] Error 1
krb5 failed to build: `make V=0 -j4` exited with 2
```

--- a/packages/c_ares.rb
+++ b/packages/c_ares.rb
@@ -7,6 +7,18 @@ class C_ares < Package
   source_url 'https://c-ares.haxx.se/download/c-ares-1.14.0.tar.gz'
   source_sha256 '45d3c1fd29263ceec2afc8ff9cd06d5f8f889636eb4e80ce3cc7f0eaf7aadc6e'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/c_ares-1.14.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/c_ares-1.14.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/c_ares-1.14.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/c_ares-1.14.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '7d03bd0a5b53a42f2c3e32f9a4dfe567f9de6b1214cd0fcfe90e35f2b93472ce',
+     armv7l: '7d03bd0a5b53a42f2c3e32f9a4dfe567f9de6b1214cd0fcfe90e35f2b93472ce',
+       i686: 'eecab4942b826db4a026b5a18bd910789cdf1558da9b8f57bf0ffff8c0d562cf',
+     x86_64: '0cc2f4240991705665aa3c6cb4ca196e54808c441e6acac983c9c278dbaae2f8',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/ctags.rb
+++ b/packages/ctags.rb
@@ -7,16 +7,28 @@ class Ctags < Package
   source_url 'http://prdownloads.sourceforge.net/ctags/ctags-5.8.tar.gz'
   source_sha256 '0e44b45dcabe969e0bbbb11e30c246f81abe5d32012db37395eb57d66e9e99c7'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ctags-5.8-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ctags-5.8-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ctags-5.8-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ctags-5.8-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd9eb4193eca3a4c7939d8e9e35ba63cbf8be084982f98c6ba0d9c4d12f3ca762',
+     armv7l: 'd9eb4193eca3a4c7939d8e9e35ba63cbf8be084982f98c6ba0d9c4d12f3ca762',
+       i686: '5afeda5261207a409a7569ec903df5c47845f237dedc9e78bd0a85bed4b780cc',
+     x86_64: 'ab996808d0c2c63d729e036cd12d5a38829dfb4bf9c5ee240db1f7201a55c723',
+  })
 
   def self.build
-   system "./configure  --prefix=#{CREW_PREFIX}  --libdir=#{CREW_LIB_PREFIX}"
-   system "make"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system "make"
   end
 
   def self.install
     #system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"  # it does not work here
-    system "install -D755 ctags #{CREW_DEST_PREFIX}/bin/ctags"
+    system "install -Dm755 ctags #{CREW_DEST_PREFIX}/bin/ctags"
     system "gzip -9 ctags.1"
-    system "install -D644 ctags.1.gz #{CREW_DEST_PREFIX}/share/man/man1/ctags.1.gz"
+    system "install -Dm644 ctags.1.gz #{CREW_DEST_PREFIX}/share/man/man1/ctags.1.gz"
   end
 end

--- a/packages/dbus.rb
+++ b/packages/dbus.rb
@@ -7,7 +7,18 @@ class Dbus < Package
   source_url 'https://dbus.freedesktop.org/releases/dbus/dbus-1.13.2.tar.gz'
   source_sha256 '945deb349a7e2999184827c17351c1bf93c6395b9c3ade0c91cad42cb93435b1'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/dbus-1.13.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/dbus-1.13.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/dbus-1.13.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/dbus-1.13.2-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd57030c6ea1f7357dc1005374f185cdb4281b2eaa65f2dfaaa977b103c06e322',
+     armv7l: 'd57030c6ea1f7357dc1005374f185cdb4281b2eaa65f2dfaaa977b103c06e322',
+       i686: 'f08b4f22f911d553019b6729958c0357c13f0bda8f075b5c4c22a1a3cb30f066',
+     x86_64: '16945f1e1ad2441e1eaf5c5ddd63b187cbda5447b75910102d6e1cfd6ffc2b3a',
+  })
 
   depends_on 'expat'
 

--- a/packages/krb5.rb
+++ b/packages/krb5.rb
@@ -8,11 +8,19 @@ class Krb5 < Package
   source_sha256 'faeb125f83b0fb4cdb2f99f088140631bb47d975982de0956d18c85842969e08'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/krb5-1.16-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/krb5-1.16-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/krb5-1.16-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/krb5-1.16-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: 'bcb8897583e93562cc1fc0c16fdb712dfa464d833c6aac83a668201763d6deeb',
+     armv7l: 'bcb8897583e93562cc1fc0c16fdb712dfa464d833c6aac83a668201763d6deeb',
+       i686: '15ce6e5b300eb7b050c3495b99aaa91c07166144783a20d9278457215bf22c52',
+     x86_64: 'fb5fb9ad6e393104a31c1ef29208840720b796555fa9f7ccd0bbd0412e44495e',
   })
+
+  depends_on 'gawk'
 
   def self.build
     Dir.chdir "src" do

--- a/packages/lcms.rb
+++ b/packages/lcms.rb
@@ -8,10 +8,16 @@ class Lcms < Package
   source_sha256 '48c6fdf98396fa245ed86e622028caf49b96fa22f3e5734f853f806fbc8e7d20'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/lcms-2.9-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/lcms-2.9-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/lcms-2.9-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/lcms-2.9-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '4e99a0236ce9f49f2a73fbc3b796112c0bc548c8bf6bb6af454e2b8c8fca5fd7',
+     armv7l: '4e99a0236ce9f49f2a73fbc3b796112c0bc548c8bf6bb6af454e2b8c8fca5fd7',
+       i686: '468dcb84ddeb0b7f4f486f766e7530e60ecdb020b5457cc65b5dbc3e5e0137d5',
+     x86_64: '47e4c1c86f4a455920106d10bfa4ec8cd47f36e41b71ea27e3bac6551a09d68c',
   })
 
   def self.build

--- a/packages/libevent.rb
+++ b/packages/libevent.rb
@@ -7,11 +7,23 @@ class Libevent < Package
   source_url 'https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz'
   source_sha256 '965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libevent-2.1.8-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libevent-2.1.8-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libevent-2.1.8-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libevent-2.1.8-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '7e6f34d7a5b630d11a28f1db646f4b1a25f4a52a16ca218de1ea888c8902fb3c',
+     armv7l: '7e6f34d7a5b630d11a28f1db646f4b1a25f4a52a16ca218de1ea888c8902fb3c',
+       i686: '766487b0654c830515d6e8759bc62ae7844b648221efa05d5a83ba2b2516a36f',
+     x86_64: '414ce2cd93f54f9a904f6ce59f4111d85cc1f126ea059750e6fe7df1fbe9eb1d',
+  })
 
   depends_on 'openssl'
 
   def self.build
-    system "./configure  --prefix=#{CREW_PREFIX}  --libdir=#{CREW_LIB_PREFIX}"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 

--- a/packages/libfdkaac.rb
+++ b/packages/libfdkaac.rb
@@ -3,17 +3,26 @@ require 'package'
 class Libfdkaac < Package
   description 'A standalone library of the Fraunhofer FDK AAC code from Android.'
   homepage 'https://github.com/mstorsjo/fdk-aac/'
-  version '0.1.6 '
+  version '0.1.6'
   source_url 'https://github.com/mstorsjo/fdk-aac/archive/v0.1.6.tar.gz'
   source_sha256 'adbcd793e406e1b88b3c1c41382d49f8c27371485b823c0fdab69c9124fd2ce3'
 
-
-
-  depends_on 'automake'
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libfdkaac-0.1.6-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libfdkaac-0.1.6-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libfdkaac-0.1.6-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libfdkaac-0.1.6-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'c75efa60c7bc827c5d14705ae4c16d22e880b748663320259118327d185b3732',
+     armv7l: 'c75efa60c7bc827c5d14705ae4c16d22e880b748663320259118327d185b3732',
+       i686: '646ee23c1ee24c7d7db11233c967c3bae3b0edef29bdb1d741fb4bcb5f563931',
+     x86_64: '6647c1bd76d4ae09c998816512be25d7b3669c1a99fb2959246102a46917ca07',
+  })
 
   def self.build
     system "./autogen.sh"
-    system "./configure  --prefix=#{CREW_PREFIX}  --libdir=#{CREW_LIB_PREFIX}"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 

--- a/packages/libjpeg.rb
+++ b/packages/libjpeg.rb
@@ -8,10 +8,16 @@ class Libjpeg < Package
   source_sha256 '1f3a3f610f57e88ff3f1f9db530c605f3949ee6e78002552e324d493cf086ad4'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libjpeg-9.0-c-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libjpeg-9.0-c-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libjpeg-9.0-c-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libjpeg-9.0-c-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '46d6bf16c5b1cc601bbf7c81de56b8451ccfe15b8660a0f61a5dc882e8f68dec',
+     armv7l: '46d6bf16c5b1cc601bbf7c81de56b8451ccfe15b8660a0f61a5dc882e8f68dec',
+       i686: 'a1afb706120d5396f34536e8613e0a1c81e927d8d0dbc6d910e03563dd1be029',
+     x86_64: 'd60d7686e7f8b5808bc5f96518da10061c507d0f1da59021d2869ff1662432b6',
   })
 
   def self.build

--- a/packages/libtirpc.rb
+++ b/packages/libtirpc.rb
@@ -8,10 +8,16 @@ class Libtirpc < Package
   source_sha256 '723c5ce92706cbb601a8db09110df1b4b69391643158f20ff587e20e7c5f90f5'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtirpc-1.0.2-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtirpc-1.0.2-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtirpc-1.0.2-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtirpc-1.0.2-0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: 'f2af84d80968a7b7271d7ed00e9fa771157530a237282c9c0cea0eea39fea167',
+     armv7l: 'f2af84d80968a7b7271d7ed00e9fa771157530a237282c9c0cea0eea39fea167',
+       i686: '3f484468cba8c5e69bff19eaa5c326dbb7e2e44d7e41c1c7ce3f570d0a7eb526',
+     x86_64: '6d4a7b558189ac4b78c24f6d5aa5904ed2dfe960ff3a6b1a4019294e2c051fc7',
   })
 
   depends_on 'krb5'
@@ -25,7 +31,6 @@ class Libtirpc < Package
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
     system "mkdir #{CREW_DEST_DIR}/usr/local/include/rpc"
     system "mv #{CREW_DEST_DIR}/usr/local/include/tirpc/rpc/* #{CREW_DEST_DIR}/usr/local/include/rpc/"
-    
     system "curl -Lo #{CREW_DEST_DIR}/usr/local/include/netconfig.h https://ftp.netbsd.org/pub/NetBSD/NetBSD-current/src/include/netconfig.h"
   end
 end

--- a/packages/libunwind.rb
+++ b/packages/libunwind.rb
@@ -7,6 +7,18 @@ class Libunwind < Package
   source_url 'http://download.savannah.gnu.org/releases/libunwind/libunwind-1.2.1.tar.gz'
   source_sha256 '3f3ecb90e28cbe53fba7a4a27ccce7aad188d3210bb1964a923a731a27a75acb'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libunwind-1.2.1-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libunwind-1.2.1-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libunwind-1.2.1-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libunwind-1.2.1-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'ad248bc364a305c449d684ba65649f0c952a0e9dcf7742411f7918346ba6ed20',
+     armv7l: 'ad248bc364a305c449d684ba65649f0c952a0e9dcf7742411f7918346ba6ed20',
+       i686: '5a5730f583590483fbac1c4a64970db358449767d2858ad01d13f1f33f3f4d86',
+     x86_64: 'd439b332a4a18ee2a49d0877409c71f28d49f539e31aec87feb6247ad4c22e41',
+  })
 
   depends_on 'buildessential' => :build
   depends_on 'openssl' => :build

--- a/packages/libxtrans.rb
+++ b/packages/libxtrans.rb
@@ -8,10 +8,16 @@ class Libxtrans < Package
   source_sha256 'b7a577c1b6c75030145e53b4793db9c88f9359ac49e7d771d4385d21b3e5945d'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxtrans-1.3.5-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxtrans-1.3.5-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxtrans-1.3.5-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxtrans-1.3.5-0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '1733f56bfe385bd68cdc430070a93f6eb6ddea511f4a994788215e00c797ca7b',
+     armv7l: '1733f56bfe385bd68cdc430070a93f6eb6ddea511f4a994788215e00c797ca7b',
+       i686: '3671105eeb2208f79fce5cdbae80722f63913d3bc13e6eb86ec41c6a5db19cf3',
+     x86_64: '63e17dadab029b422ae76939b53a0a9a38c61e654398270c34b23dd0165705db',
   })
 
   def self.build

